conf/modules.config

```diff
@@ -125,26 +130,43 @@ process {
 
     withName: "BWA.*MEM" {
         ext.args         = { meta.status == 1 ? '-K 100000000 -M -B 3' : '-K 100000000 -M' }
-        publishDir       = [ enabled: false ]
         ext.prefix       = { params.split_fastq > 1 ? "${meta.id}".concat(reads.get(0).name.findAll(/part_([0-9]+)?/).last().concat('.')) : "${meta.id}" }
+        publishDir = [
```

Comment:
if it is not enabled, i don't think this needs to be here

Reply:
I like having the same format, and if someone decides tomorrow to output/publish this module. All they need to do is to change `enabled` But it's definitely not used now